src/fsharp/NameResolution.fs

```diff
@@ -4208,17 +4212,27 @@ let ResolveCompletionsInTypeForItem (ncenv: NameResolver) nenv m ad statics ty (
             | Item.MethodGroup _ ->
                 // REVIEW: add a name filter here in the common cases?
                 let minfos = 
-                    let minfos =
-                        AllMethInfosOfTypeInScope ncenv.InfoReader nenv (None,ad) PreferOverrides m ty 
-                        |> List.filter minfoFilter
+                    let minfos = AllMethInfosOfTypeInScope ncenv.InfoReader nenv (None,ad) PreferOverrides m ty 
+                    if isNil minfos then [] else
+                     
+                    let suppressedMethNames = Zset.ofList String.order (pinfoMethNames @ einfoMethNames)
```

Comment:
@forki Could you please tell what is the advantage of using Zset here instead of HashSet?

Reply:
I did not change that part. It was zset before.

Reply:
Yes, I know. I'm just curious and thought you could know the rationale.

Reply:
I assume it's a relict from old ocaml compat times.

Reply:
I assume it's a relict from old ocaml compat times.